tests/openvino/test_exporters_cli.py

```diff
@@ -219,6 +221,40 @@ def test_exporters_cli_int4(self, task: str, model_type: str, option: str, expec
             self.assertEqual(expected_int4, num_int4)
             self.assertTrue("--awq" not in option or b"Applying AWQ" in result.stdout)
 
+    def test_exporters_cli_int4_with_local_model_and_default_config(self):
+        with TemporaryDirectory() as tmpdir:
+            pt_model = AutoModelForCausalLM.from_pretrained(MODEL_NAMES["bloom"])
+            # overload for matching with default configuration
+            pt_model.config._name_or_path = "bigscience/bloomz-7b1"
+            pt_model.save_pretrained(tmpdir)
+            result = subprocess.run(
+                f"optimum-cli export openvino --model {tmpdir} --task text-generation-with-past --weight-format int4 {tmpdir}",
+                shell=True,
+                check=True,
+                capture_output=True,
+            )
+            model_kwargs = {"use_cache": True}
+            model = eval(_HEAD_TO_AUTOMODELS["text-generation"]).from_pretrained(tmpdir, **model_kwargs)
+            rt_info = model.model.get_rt_info()
+            self.assertTrue("nncf" in rt_info)
+            self.assertTrue("weight_compression" in rt_info["nncf"])
+            default_config = _DEFAULT_4BIT_CONFIGS["bigscience/bloomz-7b1"]
+            model_weight_compression_config = rt_info["nncf"]["weight_compression"]
+            sym = default_config.pop("sym", False)
+            bits = default_config.pop("bits", 4)
+
+            mode = f'int{bits}_{"sym" if sym else "asym"}'
+            default_config["mode"] = mode
+            for key, value in default_config.items():
+                if key in ["sym", "bits"]:
+                    continue
```

Comment:
why ?

Reply:
because there is no such fields in IR meta, nncf report mode instead, here expected runtime info for reference:

```
	<rt_info>
		<Runtime_version value="2024.2.0-15496-17f8e86e5f2-releases/2024/2" />
		<conversion_parameters>
			<framework value="pytorch" />
			<is_python_object value="True" />
		</conversion_parameters>
		<nncf>
			<friendly_names_were_updated value="True" />
			<weight_compression>
				<all_layers value="False" />
				<awq value="False" />
				<group_size value="32" />
				<ignored_scope value="[]" />
				<mode value="int4_asym" />
				<ratio value="0.6" />
				<scale_estimation value="False" />
				<sensitivity_metric value="weight_quantization_error" />
			</weight_compression>
		</nncf>
		<optimum>
			<nncf_version value="2.11.0.dev0+90a7f0d5" />
			<optimum_intel_version value="1.18.0.dev0+478e981" />
			<optimum_version value="1.20.0" />
			<pytorch_version value="2.3.0" />
			<transformers_version value="4.40.2" />
		</optimum>
	</rt_info>
```

Reply:
anyway as we pop them before, this code looks unnecessary